Correct broken link in subcompositor chapter

TheSirC opened the pull request:
If the link is relative, no need to put the whole hierachy in the link.

TheSirC committed:
Correct broken link in subcompositor chapter

If the link is relative, no need to put the whole hierachy in the link.

TheSirC committed:
Continuing corrections of links

The folder is already a sibling in the directory thus no need to add it in the relative path.
Some time  inlining is the only option because of limitations of the publishing tool.

TheSirC committed:
Continuing corrections of links

Just a backslash missing giving a malformed link.
Another broken link fixed

The folder is already a sibling in the directory thus no need to add it in the relative path.
Same as previous commit


This time iniling because of limitations of the publishing tool

TheSirC committed:
Merge remote-tracking branch 'TheSirC/typo' into typo

elinorbgr commented:
What are the limitations of rustbook that forced you to in-line some links? Did something change with it?

TheSirC commented on src/wayland/structure.md:
```diff
@@ -54,12 +54,12 @@ In the end, this kind of hierachy can be expected:
 ## Protocol extensions
 
 The set of objects and the list of their requests and events is defined in
-[an XML file][wayland spec]. This approach allows wayland protocol extensions to be defined easily.
```
My bad. In momentum of correcting I just inlined this link but it _already works_. That was the limitation I was talking about (the fact that rust doc might not be able to see that it is an external link to the document).

TheSirC committed:
Correcting the correction

It seems that external link with a tag works in contrary to what I experiencied

elinorbgr commented:
Alright, thanks!
